packages/core/src/Utils/Datadog/index.ts

```diff
@@ -29,7 +29,7 @@ const getConfigValues = (environment: string) => {
         return {
             serviceName: 'staging-app.deriv.com',
             dataDogVersion: `deriv-app-staging-v${process.env.REF_NAME}`,
-            dataDogSessionReplaySampleRate: 100,
+            dataDogSessionReplaySampleRate: 0,
```

Comment:
@yaswanth-deriv, I removed it after discussing with @ali-hosseini-deriv. Because in version 4, we had to manually trigger  `startSessionReplayRecording`, which was removed in [this](https://github.com/binary-com/deriv-app/pull/12799/files#:~:text=datadogRum.startSessionReplayRecording()%3B) PR, and  `v5` has automatic recording as seen in the [change log](startSessionReplayRecording)

Reply:
Okay..Everything looks good to me.